### PR TITLE
Configure and initialize mkdocstrings

### PR DESCRIPTION
**Pull request by Laurent2916**
- Set a bunch of `mkdocstrings` options
- Remove `toc.integrate` theme feature
- Setup `literate-nav` plugin

### CODE REVIEW

File: docs/reference/SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+* <code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Fluxion
+    * [<code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Adapters](fluxion/adapters.md)
+    * [<code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Context](fluxion/context.md)
+    * [<code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Layers](fluxion/layers.md)
+    * [<code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Model Converter](fluxion/model_converter.md)
+    * [<code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Utils](fluxion/utils.md)
+* <code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Foundationals Models
```
Comment (deltheil) [inline docs/reference/SUMMARY.md:7]:
```suggestion
* <code class="doc-symbol doc-symbol-nav doc-symbol-module"></code> Foundational Models
```

File: docs/reference/foundationals/clip.md
Comment (deltheil) [inline docs/reference/foundationals/clip.md:1]: Currently, CLIP appears under `/reference/foundationals/latent_diffusion/` (see below), despite this entry. Is there a way to change that?

Screenshot 2024-01-31 at 21 26 35

Comment (catwell) [reply docs/reference/foundationals/clip.md:1]: The reason why it shows in `latent_diffusion` is it is exported by `latent_diffusion`.

The reason it does not show in `clip` is clip exports nothing.

**Review by deltheil — APPROVED** [review]

File: docs/reference/foundationals/clip.md
Comment (Laurent2916) [reply docs/reference/foundationals/clip.md:1]: Thx, I will apply this patch in later PR, let's merge the current configs

**Review by deltheil — APPROVED** [review]

**Pull request merged**